packages/workbox-routing/Route.mjs

```diff
@@ -70,9 +65,10 @@ import './_version.mjs';
  */
 
 /**
- * A `Route` consists or a matcher and a handler. A matcher needs to determine
- * if a route should be used for a request. A handler returns a response
- * to the request if there's a match.
+ * A `Route` consists of a pair of callback functions, "match" and "handler".
+ * The "match" callback determine if a route should be used to "handle" a
+ * request by returning a non-falsy value if it can. The "handler" callback
+ * is called when there is a match and should return a `Response`.
```

Comment:
This should probably say "promise for a `Response`" or "resolves to a `Response`".